packages/xo-web/src/common/intl/messages.js

```diff
@@ -592,6 +592,10 @@ const messages = {
   remotePlaceHolderPassword: 'Password(fill to edit)',
 
   // ------ New Storage -----
+
+  newSr: 'New SR',
+  newSrConfirm:
+    'This will erase the entire disk or partition to create a new storage repository. Are you sure you want to continue?',
```

Comment:
It is possible to add the name of the concerned disk ?

Reply:
I also think the name can be useful, since there can be multiple disk, with some locals and other remotes